### PR TITLE
date range picker implemented using directives

### PR DESCRIPTION
**Pull request by dev-genesistechnologies**
Before you submit a pull request, please make sure:

- [ ] The pull request should include a description of the problem you're trying to solve
- [ ] The pull request should include overview of the suggested solution
- [ ] If the pull requests changes current behavior, reasons why your solution is better.
- [ ] The proposed code should be fully functional
- [ ] The proposed code should contain tests relevant to prove is functionality
- [ ] The proposed tests should ensure significative code coverage
- [ ] All new and existing tests should pass

### CODE REVIEW

**Comment by evilaliv3** [comment]
Very good work @dev-genesistechnologies 

I noticed very little defects and i consider this entire work is ready for integration

On improvement that i would consider adding is to give the color #377ABC to every icon when the filter is active.
This would help users identify which are the activated filters. Without this i see in fact that one can activate multiple filter and see no data and forget which filter they activated.

Do you mind adding this last correction?

**Comment by dev-genesistechnologies** [comment]
> 

sure will go through it in the morning

**Comment by dev-genesistechnologies** [comment]
> Very good work @dev-genesistechnologies
> 
> I noticed very little defects and i consider this entire work is ready for integration
> 
> On improvement that i would consider adding is to give the color #377ABC to every icon when the filter is active. This would help users identify which are the activated filters. Without this i see in fact that one can activate multiple filter and see no data and forget which filter they activated.
> 
> Do you mind adding this last correction?

@evilaliv3

**Comment by evilaliv3** [comment]
Thank you @dev-genesistechnologies . Well done!

I think we could proceed integrating your work.

I will proceed mergin your pull inside the branch feature/datafilters; then i will apply some cleanhup of the branch to help who would eventually need to audit the feature as a whole and then merge it in devel preserving your credits.

File: client/app/js/directives.js [hunk 1]
```diff
@@ -59,8 +63,18 @@ GL
...
+         if ($scope.daterangePickerModel.start && $scope.daterangePickerModel.end) {
+           $scope.obj.value1 = true;
+         }else{
+           $scope.obj.value1 = false;
+         }
+
          $scope.onDatechange($scope.dataRangeFilter);
       });
+      $scope.checkFilter = function() {
+        return $scope.obj.value1 === true;
```
Comment (evilaliv3) [inline client/app/js/directives.js]: @dev-genesistechnologies : i will simply remove this variable and change this function to do:

return "return $scope.daterangePickerModel.start && $scope.daterangePickerModel.end;"

**Pull request merged**